digiroad2-oracle/src/main/resources/db/migration/V122__update_link_status_in_project_link_according_to_TR.sql

```diff
@@ -0,0 +1,8 @@
+UPDATE PROJECT_LINK SET STATUS = CASE STATUS WHEN 1 THEN 5
+                                             WHEN 2 THEN 2
+                                             WHEN 3 THEN 3
+                                             WHEN 4 THEN 1
+                                             WHEN 5 THEN 4
+                                             WHEN 0 THEN 0
+                                             ELSE 99
```

Comment:
I think here also would be better with the enum types

Reply:
Within SQL migration script we don't have access to them.

Reply:
Right, didn't noticed it was a migration script, sorry!